airbyte-integrations/connector-templates/source-python/source_{{snakeCase name}}/source.py.hbs

```diff
@@ -111,7 +111,7 @@ class Customers({{properCase name}}Stream):
 
 
 # Basic incremental stream
-class Incremental{{properCase name}}Stream({{properCase name}}Stream, ABC):
+class Incremental{{properCase name}}Stream({{properCase name}}Stream, StateMixin, ABC):
```

Comment:
will this work?

It's currently still called `IncrementalMixin` in [master](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/streams/__init__.py). But the plan is to rename this from `IncrementalMixin` to `StateMixin` once this [37429](https://github.com/airbytehq/airbyte/pull/37429) gets merged